ingredient/ingredient-app-insights/package-lock.json

```diff
@@ -0,0 +1,954 @@
+{
+  "name": "@azbake/ingredient-app-insights",
+  "version": "0.0.1",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+<<<<<<< HEAD
```

Comment:
This needs to be fixed

Reply:
@bdschaap Resolve this lock file and I can get this merged, released today.

Reply:
fixed (regenerated) the lock file